.github/workflows/build-manywheel-images.yml

```diff
@@ -137,18 +137,3 @@ jobs:
       - name: Build Docker Image
         run: |
           manywheel/build_docker.sh
-  build-docker-cpu-s390x:
```

Comment:
Is this a correct understanding that we can now remove this and let Docker pull figure out the correct CPU arch (s390x) corresponding to this change https://github.com/pytorch/pytorch/pull/120398/commits/2928b2c4d1bfa0ac03881b87cb965432ac381da2?

Reply:
No, since I couldn't find where I could download built image for s390x, I just built it manually and backed it into my github actions runner docker image. So it's available locally and isn't needed to be built. I'm also fine with actually building it here and downloading it by actions runner, if it's publicly available somewhere.